backend/api/models/follower.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+import uuid
+
+# Related Model
+from api.models.author import Author
+
+class Follower(models.Model):
+    # Unique ID of this follower Record
+    follow_id  = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    
+    # The person who want to follow
+    follower   = models.ForeignKey(Author, null=False, on_delete=models.CASCADE, related_name="follower")
+
+    # The person who is being followed( The person who should get the request )
+    followee   = models.ForeignKey(Author, null=False, on_delete=models.CASCADE, related_name="followee")
+    
+    # Not sure if needed but added it just in case
+    accepted   = models.BooleanField(default=False)
```

Comment:
I dont think we need it `accepted   = models.BooleanField(default=False)`, if friend request get accepted, the user goes to friend table. All uses in table follower should be users that haven't get accepted